compiler/src/dmd/cpreprocess.d

```diff
@@ -120,6 +120,12 @@ private const(char)[] cppCommand()
             VSOptions vsopt;
             vsopt.initialize();
             const path = vsopt.compilerPath(target.isX86_64);
+            //if the path to cl.exe is found, check if cl.exe is in the path.
+            if(FileName.exists(path) != 1)
+            {
+                error(Loc.initial, "Error: cl.exe not found. Ensure that Visual Studio is installed and properly configured");
```

Comment:
```suggestion
                error(Loc.initial, "Error: cl.exe not found. Please ensure that Visual Studio Build Tools are installed and properly configured.");
```

Reply:
There’s no need for a full VS installation. The “Visual Studio Build Tools” bundle is enough.